BUILD.bazel

```diff
@@ -0,0 +1,51 @@
+cc_library(
+    name = "dd_opentracing_cpp",
+    srcs = glob(["src/*.cpp", "src/*.h"]) + [
+        ":version_number.h",
+    ],
+    hdrs = [
+        "include/datadog/opentracing.h",
+    ],
+    strip_include_prefix = "include",
+    visibility = ["//visibility:public"],
+    deps = [
+        "//:3rd_party_nlohmann",
+        "@io_opentracing_cpp//:opentracing",
+        "@com_github_msgpack_msgpack_c//:msgpack",
+    ],
+    copts = ["-std=c++14"],
+)
+
+genrule(
```

Comment:
ahaha this was such cmake-specific bastardry that you had to replicate 😅

Reply:
Yup, inspired by lightstep's bazel setup.